### DISCUSSION TITLE
Suggesting of existing elements

### DISCUSSION THREAD

**Discussion by jmigual** [community/community#39068] [Copilot Conversations]
### Select Topic Area

Bug

### Body

More often than not, GitHub Copilot suggests lines that are already there. For example in the following example: 
![image](https://user-images.githubusercontent.com/6493651/201663717-68c17899-18a1-4a67-90e3-66803100b130.png)

The suggestion of `Delay` is correct but the suggestion of `*/` is incorrect as that value is already in the code. I found other instances of this issue already happening. My solution for now is to accept the suggestion and then delete the repeated lines. 

Has anyone else also found this issue? Can it be mitigated somehow?

**Comment by iamcjwilliams** [comment]
@jmigual Which IDE are you using?

**Reply by jmigual** [reply]
VS Code with the latest update. I'm on mobile now but if you want I can give you more plugin and editor versions later.

**Reply by iamcjwilliams** [reply]
Our copilot team is investigating this issue

**Reply by jmigual** [reply]
Just as an update I'm currently using VS Code version 1.73.1 and GitHub Copilot version v1.59.7338 and this is still happening. The language I was using is C++
